roles/common/tasks/kernel-tuning.yml

```diff
@@ -28,6 +28,13 @@
               line="GRUB_CMDLINE_LINUX_DEFAULT=\"\""
   notify: update grub config
 
+- name: "Limit available cores via grub"
+  lineinfile: dest=/etc/default/grub
+              regexp="^GRUB_CMDLINE_LINUX_DEFAULT="
+              line="GRUB_CMDLINE_LINUX_DEFAULT=\"{{ cpulimit }}\""
```

Comment:
This should just be added to the set of boot parameters on the task `Kernel boot parameters:`

To sort out how to add this conditionally, you'll want to add this to the `line=` at the end:

`{{- ' maxcpu=%s' % maxcpu if maxcpu is defined else '' }}`

The spacing is important, as is the `-` after the opening `{{`.